### DISCUSSION TITLE
Tolerations for Listener pod

### DISCUSSION THREAD

**Discussion by ealasgarov** [actions/actions-runner-controller#2942] [Questions] [answered]
Hello,
I cannot find where to specify tolerations for the listener pod. For the "gha-runner-scale-set-controller" I've specified the tolerations in the values yaml file and it is working for "arc-gha-rs-controller" pods/replica, but not for the listener pod. I also specified tolerations for the runner pod in 'gha-runner-scale-set' value yaml. Just cannot locate yet where should this be done for the listener... please assist. 
Thanks in advance!

**Comment by nikola-jokic** [comment, marked as answer]
You can customize the listener pod using [this field](https://github.com/actions/actions-runner-controller/blob/1a8abb6d3950d3a78715505312d88b201e6d8a49/charts/gha-runner-scale-set/values.yaml#L90C10-L90C10) in `gha-runner-scale-set` chart, starting from version 0.6.0 :relaxed:.

However, it should be pointed out that 0.6.1 version should be used because 0.6.0 version contains a bug when modifying listener annotations

**Reply by ealasgarov** [reply]
thank you!
